eda-env/Lib/site-packages/pandas/tests/window/test_timeseries_window.py

```
@@ -0,0 +1,715 @@
+    def test_doc_string(self):
+        df = DataFrame(
+            {"B": [0, 1, 2, np.nan, 4]},
+            index=[
+                Timestamp("20130101 09:00:00"),
+                Timestamp("20130101 09:00:02"),
+                Timestamp("20130101 09:00:03"),
+                Timestamp("20130101 09:00:05"),
+                Timestamp("20130101 09:00:06"),
+            ],
+        )
+        df
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)